pkg/kubelet/kubelet_pods_test.go

```diff
@@ -1711,39 +1710,6 @@ func TestGetHostPortConflicts(t *testing.T) {
 	assert.True(t, hasHostPortConflicts(pods), "Should have port conflicts")
 }
 
-func TestMakeDevices(t *testing.T) {
```

Comment:
Yeah. Unit tests are pending. I will be adding quite a few of them soon, may be not before the code freeze. Bear with me.